be/src/runtime/thread_mem_tracker_mgr.h

```diff
@@ -175,39 +170,70 @@ class ThreadMemTrackerMgr {
     ConsumeErrCallBackInfo _consume_err_cb;
 };
 
+inline void ThreadMemTrackerMgr::init() {
+    _tracker_id = 0;
+    _mem_trackers.clear();
+    _mem_trackers[0] = MemTracker::get_process_tracker();
+    _untracked_mems.clear();
+    _untracked_mems[0] = 0;
+    _mem_tracker_labels.clear();
+    _mem_tracker_labels[0] = MemTracker::get_process_tracker()->label();
+}
+
+inline void ThreadMemTrackerMgr::init_bthread() {
+    init();
+    _mem_trackers[1] = MemTracker::get_brpc_server_tracker();
+    _untracked_mems[1] = 0;
```

Comment:
Is it safe to use 0 and 1 as the tracker id for process tracker and brpc tracker?

Reply:
Safe because the map key is the tracker id
id = (GetCurrentTimeMicros() % 1000000) * 100 + _label.length()

(Actually, I'm not 100% sure if the id could be duplicated... but it should be faster than converting to int after hash)